src/gui/properties/peerlistwidget.cpp

```diff
@@ -458,14 +458,16 @@ void PeerListWidget::handleSortColumnChanged(int col)
 
 void PeerListWidget::wheelEvent(QWheelEvent *event)
 {
-    event->accept();
-
     if (event->modifiers() & Qt::ShiftModifier) {
         // Shift + scroll = horizontal scroll
+        event->accept();
+
         QWheelEvent scrollHEvent(event->pos(), event->globalPos(), event->delta(), event->buttons(), event->modifiers(), Qt::Horizontal);
         QTreeView::wheelEvent(&scrollHEvent);
         return;
     }
 
-    QTreeView::wheelEvent(event);  // event delegated to base class
+    // event delegated to base class
+    event->ignore();
```

Comment:
>This is mentioned explicitly in Qt doc:
If you reimplement this handler, it is very important that you ignore()
the event if you do not handle it, so that the widget's parent can
interpret it.

Seems like you misinterpreted Qt docs...
widget's parent != widget's base class
You don't need to call `ignore()` (or `accept()`) immediatelly before call to base event handler (`QTreeView::wheelEvent(event);`) since this flag will be handled there. You only need to do it if you don't delegate event to base class handler.

Reply:
Thanks! I've updated the commit.